packages/fx-core/package.json

```diff
@@ -101,6 +101,7 @@
     "node-forge": "^1.0.0",
     "node-ts-uuid": "^1.0.8",
     "office-addin-manifest": "^1.10.4",
+    "office-addin-project": "0.5.0",
```

Comment:
Please commit package-lock.json file as well, or CI/CD pipeline would fail. Becasue `npm ci` checks whether package.json and lock are in sync.

Reply:
When I install the lock file gets udpate from version 1 format to version 2.  Is this OK?

FYI . . . This happen because I'm using an newer version of npm that goes with node 16 (which is a year old already).  The current version of npm actually has another update to the lock file version (v3).  I'm not sure what your policy is for supporting different versions of node and npm.

Reply:
We are using node v14.17.0 (npm v6.14.17), which uses lockfersion 1. I think it better to keep it that way.
You can easily switch to node 14 using nvm or other nodejs version manager.

Reply:
OK . . . I think I got the right setup and submitted the updated to the file.

Reply:
Could you please to resolve the conflict?